File: Jack.Presentation/Controllers/API/DataReuniaoController.vb
Imports System.Net
Imports System.Web.Http

Namespace Controllers.API
    Public Class DataReuniaoController
        Inherits ApiController

        <HttpGet>
        Public Function GetValues(intAno As Integer) As IList(Of Model.Reuniao)
            Dim oReuniao As Business.Reuniao = New Business.Reuniao()
            Dim oReturn As List(Of Model.Reuniao)

            oReturn = oReuniao.LoadByAnoCorrente(intAno)
            oReuniao = Nothing

            Return oReturn

        End Function


    End Class
End Namespace

File: Jack.Presentation.Web/Controllers/API/Reunião/DataReuniaoController.cs
using System;
using System.Collections.Generic;
using System.Web.Http;
using Business = Jack.Business;
using Model = Jack.Model;


namespace Controllers.API
{
	public class DataReuniaoController : ApiController
	{

		[HttpGet()]
		public IList<Model.Reuniao> GetValues([FromUri()] int ID)
		{
			Business.Reuniao oReuniao = new Business.Reuniao();
			IList<Model.Reuniao> oReturn = default(List<Model.Reuniao>);

			oReturn = oReuniao.LoadByAnoCorrente(ID);
			oReuniao = null;

			return oReturn;

		}


	}
}
